### ISSUE TITLE
setCompletionCallback?

### ISSUE THREAD

**Issue by jgassie** [mehaase/js-typewriter#1]
Hello, thanks for sharing, works well.
Not sure if this is an issue... I can't get the completion callback to run for some reason.. no errors in the console log though.. I haven't really tried to debug it yet.  Perhaps it's not meant to work when using character callback.. instead I put a regular callback in the last character sequence, and that worked fine.  I have an outdated personal site (mostly family history & such) I am toying around with possibly using this library with.  I found another js library like this out there, but I liked the ability to use the character callback for the audio, which the other did not have. Thanks again!
update: well i see the playsequenceatindex basically overwrites whatever callback value I would have set with the setCompletionCallback.

**Comment by mehaase** [comment]
Yes, sorry, the callback gets overwritten. It's not a good design :( If you happen to improve it yourself, I would be happy to merge your change.

**Comment by jgassie** [comment]
Should line 28 refer to this._textNode? 
I wanted a way to clear the text without re-creating the typewriter object... by letting the library do the work, was just playing around with it... if I used jquery to blank the object text, the typewriter object no longer had it's text node to work with.
`Typewriter.prototype.clearText = function () {
    this._textNode.nodeValue = "";
}
Typewriter.prototype.backspaceAll = function(tdelay) {
    var that = this;
    var len = this._textNode.nodeValue.length;
    if(len==0) { return;}
    this._textNode.nodeValue = this._textNode.nodeValue.substring(0, (len-1));
    setTimeout(function() { that.backspaceAll(tdelay); }, tdelay);
}`
